.github/workflows/release-sdk.yml

```diff
@@ -0,0 +1,45 @@
+# When a new release is created on GitHub, this workflow builds the SDK as an XCFramework,
+# generates the API documents, and pushes a CocoaPods update.
+name: Radar SDK iOS Release
+
+on:
+  # Triggered when a new release is tagged in GitHub.
+  release:
+    types: [ published ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build, test, and analyze
+        run: xcodebuild clean build analyze test -workspace Example/Example.xcodeproj/project.xcworkspace -scheme RadarSDK -destination "platform=iOS Simulator,name=iPhone 11" | xcpretty
+      
+      - name: Build archive for iPhone simulator
+        run: xcodebuild archive -scheme RadarSDK -archivePath "RadarSDK-iphonesimulator.xcarchive" -sdk iphonesimulator SKIP_INSTALL=NO CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO
+      
+      - name: Build archive for iPhone device
+        run: xcodebuild archive -scheme RadarSDK -archivePath "RadarSDK-iphoneos.xcarchive" -sdk iphoneos SKIP_INSTALL=NO CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO
+
+      - name: Build XCFramework
+        run: xcodebuild -create-xcframework -framework RadarSDK-iphonesimulator.xcarchive/Products/Library/Frameworks/RadarSDK.framework -framework RadarSDK-iphoneos.xcarchive/Products/Library/Frameworks/RadarSDK.framework -output RadarSDK.xcframework
+
+      - name: Zip XCFramework
+        run: zip -r RadarSDK.xcframework.zip RadarSDK.xcframework -x ".*" -x "__MACOSX" -D
+
+      - name: Generate SDK docs
+        run: jazzy
+
+      - name: Zip SDK docs
+        run: zip -r docs.zip docs -x ".*" -x "__MACOSX" -D
+      
+      - name: Lint CocoaPods spec
```

Comment:
Would it make sense for the linter to run first, in case errors are found?

Reply:
This step lints the CocoaPods spec file before we publish it; it's not a general linter like SwiftLint. I suppose it can be run earlier in the process, but again, I'm following the order in which `make dist` does things.